etcdctl/ctlv3/command/printer_simple.go

```diff
@@ -188,6 +188,12 @@ func (s *simplePrinter) RoleAdd(role string, r v3.AuthRoleAddResponse) {
 
 func (s *simplePrinter) RoleGet(role string, r v3.AuthRoleGetResponse) {
 	fmt.Printf("Role %s\n", role)
+	if "root" == role {
```

Comment:
This statement looks redundant？

Reply:
yep, the logic be handle at server side, but I'm not sure the server logic is right or not, need your guidance, thanks.

Reply:
I think it can be deleted, you can test it yourself.